hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -381,7 +381,15 @@ public static HoodieTableMetaClient initTableAndGetMetaClient(Configuration hado
     return metaClient;
   }
 
-  // HELPER METHODS TO CREATE META FILE NAMES
+  /**
+   * Helper methods to create meta file names.
```

Comment:
`methods` -> `method`. IMO, the original comment is incorrect. You should correct the java doc based on the logic of this method.

Reply:
@yanghua Ok

Reply:
@yanghua  PTAL